### DISCUSSION TITLE
Can't find how to re-authenticate

### DISCUSSION THREAD

**Discussion by JonasFovea** [abraunegg/onedrive#2664] [General]
Hello!

I've been using this onedrive client for a while now and it worked perfectly.
Now it seems as if my authentication has expired.

How do I restart the authentication process?
I did not find this in the docs, but I've probably just missed it.

Thanks in advance!

**Comment by JonasFovea** [comment]
Found it!
Simply use the --reauth option

**Reply by abraunegg** [reply]
This is also documented here: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#handling-a-onedrive-account-password-change
